extras/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Go to manifold/extras directory and run the command as `./run.sh ../build/extras/testHullPerformance {path_to_dataset_folder} {name_of_csv} {implementation(Hull,Hull_CGAL)}`
```

Comment:
Ideally, let the code take one file, and use the shell script to iterate over everything in the folder and write to a csv. This way we can quickly run the tool with a single file.

Reply:
The code takes the folder containing the files as an input and iterates over all the files in that folder. This is even simpler than specifying a file right? I can remove name of the csv parameter, if needed. The reason I kept implementation as an option is so that the user doesn't have to go and change the code to compare different implementations.

Reply:
You can do the iteration in the bash script.

```bash
for f in dataset/*; do
  ../build/extras/testHullPerformance $f Hull >> output.csv
done
```

Something like this.

Reply:
So we just assume that the input files that we wanna test will always be placed in the same dataset folder, and thus no need to take the folder for dataset as input right?

Reply:
No, the shell script expects the same dataset folder, but the actual binary just takes a single file as input.